### PR TITLE
update parallel

### PR DESCRIPTION
**Pull request by mszhanyi**
1. As dataloader, set number of worker as 0 in Notebook and multiple workers works in scripts
2. override system default encoding to UTF-8 on Windows

### CODE REVIEW

**Comment by mszhanyi** [comment]
@jph00 https://github.com/fastai/fastai/pull/3198 depends on this.

File: nbs/03_xtras.ipynb [hunk 1]
```diff
@@ -41,6 +41,17 @@
     "from time import sleep"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if sys.platform == \"win32\":\n",
```
Comment (jph00) [inline nbs/03_xtras.ipynb:50]: Looks like this cell isn't exported, so it's not doing anything.

Although I don't think this is a great solution. It's both relying on a private python implementation detail, and it's also changing the behavior of python outside of use of the library.

**Review by jph00 — CHANGES_REQUESTED** [review]
Just one request here

File: nbs/03_xtras.ipynb (hunk 1 above)
Comment (mszhanyi) [reply nbs/03_xtras.ipynb]: @jph00 Thank your comments.
similar change made in fastai too. https://github.com/fastai/fastai/pull/3203

**Pull request merged**